File: Codechef/Codechef_OCT_CookOff_2020/codechef.01.py

test = int(input())
for _ in range(test):
    n = int(input())
    str = input()

    flag = False 
    
    if n % 2 != 0:
        n += 1

    if str[-1] in str[:-1]:
        flag = True

    if flag == False:
        print("NO")
    else:
        print("YES")

File: Codechef/2020/Codechef_SEP_Lunchtime_2020/codechef.01.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int test;
    cin >> test;
    while(test--)
    {
        int n, temp, i, flag = 0;
        cin >> n;
        vector<int> v;
        for (i = 0; i < n; i++)
        {
            cin >> temp;
            v.push_back(temp);
        }
        int test_sum = accumulate(v.begin(), v.end(), 0);
        if(test_sum > 0 )
        {
            flag = 1;
        }  
        if(flag == 1)
        {
            cout << "YES" << endl;
        }
        else 
        {
            cout << "NO" << endl;
        }
    }
return 0;
}
